features/contact/requesting_contact.feature

```diff
@@ -8,7 +8,7 @@ Feature: Requesting contact
         Given the store operates on a single channel in "United States"
         And this channel has contact email set as "contact@goodshop.com"
 
-    @ui @email
+    @ui @api @email
     Scenario: Requesting contact as a logged in customer
         Given there is a user "lucifer@morningstar.com"
         And I am logged in as "lucifer@morningstar.com"
```

Comment:
I am not sure if it should be handler in different way for logged it user by API

Reply:
Probably, if the shop user is logged in, they shouldn't have to specify the email, but they can still do it if they want to 🤔